services/subscriber.py

```diff
@@ -20,11 +20,11 @@ def get_subscribers(self, targetings, hours_whitelist, volume):
         if timezones:
             targetings.append({
                 "field": "timezone",
-                "operator": "AND",
+                "operator": "IN",
                 "value": timezones
             })
         try:
-            s = Search(using=es, index="subscribers")
+            s = Search(using=es, index="users")
             s.extra(size=volume, boost_mode='replace')
```

Comment:
to set a `size` we do 

```
s = s[volume]
```

http://elasticsearch-dsl.readthedocs.io/en/latest/search_dsl.html?highlight=size#pagination